src/reactors/pr/add_label_reactor.ts

```diff
@@ -0,0 +1,63 @@
+import { LabelTransform } from '../'
+import { RELEASE_BRANCH_RE } from '../../lib'
+import { prAddLabelTransforms as presentationTeamTransforms } from '../../teams/presentation_team'
+import { ReactorInput, PrReactor } from './pr_reactor'
+
+/**
+ * Teams and other interested parties should add their label transform functions to this
+ * list.
+ *
+ * ORDER MATTERS.
+ *
+ * The functions are run in order-- left-to-right-- thus transforming the result of the previous
+ * transform.
+ */
+const labelTransforms: LabelTransform[] = [...presentationTeamTransforms]
+
+const RELEVANT_ACTIONS_MISSING_LABEL: ReactorInput['action'][] = [
+  'labeled',
+  'unlabeled',
+  'opened',
+  'synchronize',
+  'refresh',
+  'ready_for_review',
+]
+
+export const addLabelReactor = new PrReactor({
+  id: 'prAddLabelReactor',
+
+  filter: ({ input: { action, pr } }) =>
+    pr.state === 'open' &&
+    !pr.draft &&
+    RELEVANT_ACTIONS_MISSING_LABEL.includes(action),
+
+  async exec({ input: { pr, action }, githubApi, log }) {
+    log.info(`pr #${pr.number} [action=${action}]`, { action })
+    const labelNames = pr.labels.map(label => label.name)
+    let labels = [...labelNames]
+
+    labelTransforms.forEach(check => {
+      labels = check(labels)
+    })
```

Comment:
```suggestion
    const labels = labelTransforms.reduce((acc, transform) => transform(acc), [...labelNames])
```

Reply:
Nice, I can never get `reduce` straight in my head for this kind of task.